### PR TITLE
feat - added cart management page

### PR DESCRIPTION
**Pull request by akash-1318**
Screenshot 2022-02-12 at 1 01 00 AM

Added the cart management page using the vertical card from component library.

### CODE REVIEW

File: pages/styles/mycart.css [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    background-color: white;
+    border: 1px solid var(--go-to-cart);
+    color: var(--go-to-cart);
+}
+
+.mycart__product-right{
+    align-self: flex-start;
+    padding: 2rem;
+    width: 35rem;
+    background-color: white;
```
Comment (arjunbharti) [inline pages/styles/mycart.css:102]: Use color variables

File: pages/styles/mycart.css [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+.place__order-btn{
+    width: 100%;
+}
+
+.card__cs.mycart{
+    width: 55rem;
+    padding: 0px;
+    margin-top: 0px;
+    background-color: white;
```
Comment (arjunbharti) [inline pages/styles/mycart.css:140]: Use color variables

**Review by arjunbharti — COMMENTED** [review]
Just a little suggestion -  Use color variables. Everything else is fine.

**Review by sahiba0915 — COMMENTED** [review]
looks good to merge

File: pages/styles/mycart.css (hunk 1 above)
Comment (akash-1318) [reply pages/styles/mycart.css:102]: Done

**Pull request merged**